pkg/devcontainer/setup/lifecyclehooks.go

```diff
@@ -96,9 +97,13 @@ func run(commands []types.LifecycleHook, user, dir string, remoteEnv map[string]
 
 		for k, c := range cmd {
 			log.Infof("Run command %s: %s...", k, strings.Join(c, " "))
+			currentUser, err := user.Current()
+			if err != nil {
+				log.Fatalf("Error fetching the current user: %v", err)
```

Comment:
Can you return the error instead of panicing so we can gracefully shutdown the reset of the app?

Reply:
Done, thanks